app/models/rule.rb

```diff
@@ -118,4 +118,8 @@ def latest_result(host, profile)
 
     test_result.rule_results.find_by(rule_id: id)
   end
+
+  def rule_group_ancestor_ids
+    rule_group&.path_ids
+  end
```

Comment:
Do we really need a separate method for this?

Reply:
Probably not.. I can try to move it into the `tailoring_valid?` and see if rubocop complains about it

Reply:
I was able to move this line into `tailoring_valid?` and remove this method 👍